### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Layrovell**
[DEMO LINK](https://Layrovell.github.io/react_movies-list/)

### CODE REVIEW

**Review by xipholena — APPROVED** [review]
Класнаа робота!

File: src/components/MoviesList/MoviesList.js [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+    <div className="movies" key={el.imdbId}>
+      <MovieCard
+        title={el.title}
+        description={el.description}
+        imgUrl={el.imgUrl}
+        imdbUrl={el.imdbUrl}
+        imdbId={el.imdbId}
+      />
+    </div>
+  ));
```
Comment (aliceblunt8) [inline src/components/MoviesList/MoviesList.js:16]: add this to the return statement, no need to create a separate var for it

Comment (aliceblunt8) [reply src/components/MoviesList/MoviesList.js:16]: use spread operator to pass data
https://medium.com/@kevinyckim33/jsx-spread-operator-component-props-meaning-3c9bcadd2493

Comment (aliceblunt8) [reply src/components/MoviesList/MoviesList.js:16]: add key to the MovieCard itself, div is extra

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]

File: src/components/MoviesList/MoviesList.js (hunk 1 above)
Comment (aliceblunt8) [reply src/components/MoviesList/MoviesList.js:16]: propTypes?

File: src/components/MoviesList/MoviesList.js [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
-);
+export const MovieList = ({ movies }) => {
+  const MoviesArray = movies.map(movie => (
+    <div className="movies">
+      <MovieCard
+        key={movie.imdbId}
+        movie={movie}
+      />
+    </div>
+  ));
```
Comment (aliceblunt8) [inline src/components/MoviesList/MoviesList.js:15]: move this to the return statement

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**